gradle/integTest.gradle

```diff
@@ -0,0 +1,67 @@
+// From https://raw.githubusercontent.com/spring-io/sagan/master/gradle/integTest.gradle
```

Comment:
Please use https://github.com/unbroken-dome/gradle-testsets-plugin instead of copying and pasting gradle!!